packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -35,6 +35,18 @@ def expected_datapoint_response(response):
         """
         assert 'datapoints' in response, '"datapoints" dictionary not found'
         'in response, got {}'.format(response)
+
+        for dp in response['datapoints']:
+            assert 'name' in dp, '"name" parameter should not be empty, got {}'.format(dp)
```

Comment:
nit: misleading error message, is says "name" should not be empty, but you check if response['datapoints'] has field name.